hibernate-core/src/main/java/org/hibernate/transform/AliasToBeanConstructorResultTransformer.java

```diff
@@ -1,74 +1,74 @@
-/*

```

Comment:
In this file also the indentation or line endings changed. Intentionally?

Reply:
Not intentionally. It seems github abandoned the diff effort when too many files are included. Maybe after some timeout?